README.md

```diff
@@ -44,6 +44,20 @@ or using alias from `.cargo/config.toml`
 cargo i
 ```
 
+## Running Directly from source without install
+
+These commands allow you to run the tool directly from the source code without installation. For more options see [generate help](#cargo-leet-generate---help)
+
+```sh
+cargo run --features=tool -- leet gen
+```
+
+or using alias from `.cargo/config.toml`
+
+```sh
+cargo lg
```

Comment:
not correct as you will need to provide path in this case.

Reply:
That's one option. You can provide it after the lg if you really wanted to.  But when I'm testing I usually don't. I just let it work on the current repo. And then revert the changes.

Reply:
add the path part as discussed and merge this.